Fastpack/Error.ml

```diff
@@ -1,11 +1,58 @@
 module Loc = FlowParser.Loc
 module Scope = FastpackUtil.Scope
 
+type color = Cyan | Red | Black | White;;
+type font = Regular | Bold;;
+
+let print_with_color ?font:(font=Regular) str col = 
+  let col = match col with 
+    | Cyan -> "36"
+    | Red -> "31"
+    | Black -> "30"
+    | White -> "37"
+  in
+  let f = match font with 
+    | Regular -> "0"
+    | Bold -> "1"
+  in
+  "\027[" ^ f ^ ";" ^ col ^ "m" ^ str ^ "\027[0m"
+
+let get_codeframe (loc: Loc.t) lines = 
+  let startLine = max 0 (loc.start.line - 2) in
+  let endLine = min (List.length lines) (loc._end.line + 1) in
+  let codeframe = (List.filter_map (fun (i, line) -> 
```

Comment:
Outer-most parenthesis are obsolete, i.e. `let codeframe = List.filter_map ... lines in` would work fine

Reply:
done